prinz/src/test/scala/pl/touk/nussknacker/prinz/model/mlflow/MLFContainerTest.scala

```diff
@@ -0,0 +1,52 @@
+package pl.touk.nussknacker.prinz.model.mlflow
+
+import pl.touk.nussknacker.prinz.UnitIntegrationTest
+import pl.touk.nussknacker.prinz.mlflow.MlflowConstants.DEFAULT_URL
+import pl.touk.nussknacker.prinz.mlflow.repository.MLFRepository
+import pl.touk.nussknacker.prinz.model.{SignatureName, SignatureType}
+
+class MLFContainerTest extends UnitIntegrationTest {
+
+  "Mlflow container" should "list some models" in {
+    val repository = MLFRepository(DEFAULT_URL)
+    val models = repository.listModels.toOption
+
+    assert(models.isDefined)
+    assert(models.exists(_.nonEmpty))
+  }
+
+  it should "have model instance available" in {
+    val instance = getModelInstance
+
+    assert(instance.isDefined)
+  }
+
+  it should "have model instance that have signature defined" in {
+    val instance = getModelInstance
+    val signature = instance.map(_.getSignature)
+
+    assert(signature.isDefined)
+  }
+
+  it should "have specific for tests model signature" in {
+    val instance = getModelInstance
+    val signature = instance.map(_.getSignature).get
+
+    assertResult(1)(signature.getOutputType.size)
+    assertResult(SignatureType("double"))(signature.getOutputType.head)
+
+    assertResult(3)(signature.getInputDefinition.size)
+    assert(signature.getInputDefinition.contains(input("a")("double")))
+    assert(signature.getInputDefinition.contains(input("b")("double")))
+    assert(signature.getInputDefinition.contains(input("c")("double")))
```

Comment:
In tests for collection package we use matchers with "should" keyword. Let's be consistent with test style

Reply:
will check how to do it (and fix to use should) but we have te keep in mind that in the future we can use all styles of assertions as long as they works and let's say they don't introduce some new dependencies